skills/debug/references/investigation-checklist.md

```
@@ -102,17 +102,22 @@ At 15 minutes, ask:
 
-If any answer is "no" → switch to thorough track.
+If any answer is "no" -> switch to thorough track.
 
 ### Switching to Thorough Track
 
-```bash
-# Update state to switch tracks
-~/.claude/scripts/workflow-state.sh set <state-file> '.track = "thorough"'
+Use `mcp__workflow-state__workflow_set` to update state:
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)